cmd/dlt/ocmrole/cmd.go

```diff
@@ -0,0 +1,239 @@
+/*
+  Copyright (c) 2022 Red Hat, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+package ocmrole
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/spf13/cobra"
+
+	unlinkocmrole "github.com/openshift/rosa/cmd/unlink/ocmrole"
+	"github.com/openshift/rosa/pkg/aws"
+	"github.com/openshift/rosa/pkg/helper"
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/interactive/confirm"
+	"github.com/openshift/rosa/pkg/logging"
+	"github.com/openshift/rosa/pkg/ocm"
+	rprtr "github.com/openshift/rosa/pkg/reporter"
+)
+
+var args struct {
+	roleARN string
+}
+
+var Cmd = &cobra.Command{
+	Use:     "ocm-role",
+	Aliases: []string{"ocmrole"},
+	Short:   "Delete ocm role.",
+	Long:    "Delete ocm role from the current AWS account.",
+	Example: ` # Delete ocm role
+rosa delete ocm-role --role-arn arn:aws:iam::123456789012:role/xxx-OCM-Role-1223456778`,
+	RunE: run,
+}
+
+func init() {
+	flags := Cmd.Flags()
+
+	flags.StringVar(
+		&args.roleARN,
```

Comment:
what is the motivation behind getting roleARN instead of role name?

Reply:
In case the role is linked we'll want to call `rosa unlink ocm-role` and pass the `roleARN` as a parameter:
`err = unlinkocmrole.Cmd.RunE(unlinkocmrole.Cmd, []string{roleARN})`

Also to be consistent with other commands. 
In the future, if we'll want to make decisions based on other information embedded in the `roleARN`, we'll have it, otherwise, we'll need to change the argument in the command.